### ISSUE TITLE
MOSS: Databus entries are not registered on MOSS

### ISSUE THREAD

**Issue by FelixMau** [open-modex/oedatamodel_api#34]
When registering Data from OEP [example](https://openenergy-platform.org/dataedit/view/model_draft/modex_tech_chp_combined_cycle) via the [API](https://modex.rl-institut.de/databus/) on the energy.databus the [dataset](https://energy.databus.dbpedia.org/felixmaur/modex/modex_tech_chp_combined_cycle/) is not registered on MOSS. 
![grafik](https://user-images.githubusercontent.com/98911066/218082355-40796acd-8d6a-4059-b74c-25719ffb6fb4.png)

**Comment by henhuy** [comment]
I can see the dataset when going to `https://moss.tools.dbpedia.org/submit-data` and put in the link to the artifact as `https://energy.databus.dbpedia.org/felixmaur/modex/modex_tech_chp_combined_cycle`.
Not sure how the OEP_Metadata search works...

**Comment by chrwm** [comment]
Is the issue still open? 
If yes, @FelixMau in which field have you annotated the data?

**Comment by FelixMau** [comment]
Yes. I annotated [onshore wind farm](https://openenergy-platform.org/api/v0/schema/model_draft/tables/modex_tech_wind_turbine_onshore/meta/) within the metadata `subject`. And cannot find the dataset on [energy.databus](https://energy.databus.dbpedia.org/felixmaur/modex/modex_tech_wind_turbine_onshore/v8) within the [MOSS search](https://moss.tools.dbpedia.org/) so far. It might be possible that I have chosen some wrong settings but I think I have tried all combinations.

**Comment by chrwm** [comment]
What script/tool are you using to register on MOSS?
